src/pages/index.jsx

```diff
@@ -5,6 +5,7 @@ import * as React from "react";
 
 import Layout from "@/layout/Layout";
 
+import Subscribe from "../components/Subscribe";
```

Comment:
- Use @ instead of ..
For example:
- import Subscribe from "@/components/Subscribe"; ✅
- import Subscribe from "../components/Subscribe"; ❌

Reply:
Not for all imports we can use the `@`. 💡